templates/layout-slider.php

```diff
@@ -7,21 +7,20 @@
 
 <div class="gallerya gallerya--slider">
   <ul class="js-gallerya-slider js-gallerya-lightbox">
-    <?php $image_attr = Plugin::lazyLoadIsActive() ? ['data-no-lazy' => '1'] : []; ?>
+    <?php $image_attr = Plugin::isLazyLoadActive() ? ['data-no-lazy' => '1'] : []; ?>
```

Comment:
Can we append this variable to the initializations at the beginning of the file?